extensions/data-workspace/src/test/index.ts

```diff
@@ -3,46 +3,11 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as path from 'path';
+import { getDefaultMochaOptions } from '@microsoft/azdata-test';
 import * as testRunner from '@microsoft/vscodetestcover';
 
-const suite = 'Data Workspace Extension Tests';
+const options = getDefaultMochaOptions('Data Workspace Extension Tests', 'tdd');
```

Comment:
what does the `'tdd'` here mean? Other extensions like dacpac and schema compare previously had `ui: 'bdd'` but that isn't getting passed into `getDefaultMochaOptions()` in those extensions.

Reply:
It defaults to bdd : https://github.com/microsoft/azdata-test/blob/main/lib/testOptions.ts#L20

TDD vs BDD (it only really changes what the name of the test methods are - functionally it all behaves the same. https://mochajs.org/#interfaces

`The BDD interface provides describe(), context(), it(), specify(), before(), after(), beforeEach(), and afterEach()`

`The TDD interface provides suite(), test(), suiteSetup(), suiteTeardown(), setup(), and teardown()`

At some point we should really just align on one or the other to avoid confusion when looking at tests in different extensions...

Reply:
yes, I started with not exposing this and then realized that we have mixed use of tdd and bdd 😞